data_pipeline/models/github/commits/commits.sql

```diff
@@ -47,6 +47,7 @@ final as (
       c.comment_count,
       c.created_at,
       c.repo_id,
+      c.commit_org_name,
```

Comment:
You have to generate one more column and make it the new reference to users table, which no longer has user_id as the primary key.